templates/web/common/includes/display_data_quality_issues_and_improvement_opportunities.tt.html

```diff
@@ -0,0 +1,25 @@
+<!-- start templates/[% template.name %] -->
+
+[% FOREACH tagtype IN tagtypes %]
+    [% IF tagtype.product_ref_tagtype.defined && tagtype.scalar_product_ref_tagtype > 0 %]
+        <h2>[% tagtype.tagtype_heading %]</h2>
+
+        [% FOREACH tagid IN tagtype.tagids %]
+            <strong>[% tagid.display_taxonomy_tag %]</strong><br>
+
+            [% IF tagid.properties.defined %]
+                <p>[% tagid.properties %]</p>
+            [% END %]
+
+            [% IF tagtype.match('^data_quality') %]
```

Comment:
we should do this if in the template, we can do it only once in the Perl code, and simpy have a tagid.description property